### ISSUE TITLE
accessibility map: accessible places statistics do not work anymore

### ISSUE THREAD

**Issue by tahini** [chairemobilite/transition#1144]
Since the data field is not present in the nodes feature collection anymore, we can't retrieve the accessible places data (if available) for the accessibility map statistics.

#1143 fixes the accessibility map so it does not fail, but this feature should be brought back, though it should not be calculated the way it is now (ie in the frontend, from the node's data fields).

We should re-think Transition's relation with the places and how to get proper statistics when necessary.
